LICENSE-3rdparty.csv

```diff
@@ -0,0 +1 @@
+Component,Origin,License,Copyright
```

Comment:
Is this empty because it is not filled or because there is nothing to add in there?

Reply:
Nothing to add